src/main/java/org/kata/damian/PlayingCard.java

```diff
@@ -48,12 +50,16 @@ public PlayingCard(Suit suit, int value, String face) {
      */
     @Override
     public int compareTo(PlayingCard playingCard) {
-        return -1;
+        if (this.suit == playingCard.getSuit()) {
+            return Integer.compare(this.value, playingCard.getValue());
+        }
+
+        // Well this .ordinal() method is handy I must say - just saved me hours of messy code.
+        return Integer.compare(this.suit.ordinal(), playingCard.getSuit().ordinal());
```

Comment:
Haha good to see you found it, Enums also implement the Comparable interface so they should have a compareTo method that also can be used to get the natural order.